components/combined_utility/browser/combined_utility_service.cc

```diff
@@ -0,0 +1,103 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/combined_utility/browser/combined_utility_service.h"
+
+#include "components/keyed_service/content/browser_context_dependency_manager.h"
+#include "content/public/browser/service_process_host.h"
+
+template <>
+inline sandbox::mojom::Sandbox
+content::GetServiceSandboxType<combined_utility::mojom::BatAdsLedgerFactory>() {
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
this should always be sandboxed, but why do we have ads/ledger specific stuff in here anyway? I still think we're over-engineering this by not just letting rewards manage the process, but I don't think we should make this a standalone component because this isn't something we want to do in general, it's just a workaround for legacy code.